mipengine/controller/node_tasks_handler_interface.py

```diff
@@ -36,6 +36,10 @@ class IQueuedUDFAsyncResult(IAsyncResult, ABC):
 
 
 class INodeTasksHandler(ABC):
+    @abstractmethod
+    def close_app(self):
```

Comment:
minor, there is no mention of an 'app' anywhere in the interface, the method could just be named `def close(self):`

Reply:
fixed